api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -74,6 +76,25 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
         }
       };
 
+  private static final Function<org.pmiops.workbench.db.model.ConceptSet, ConceptSet> TO_CLIENT_CONCEPT_SET =
+    new Function<org.pmiops.workbench.db.model.ConceptSet, ConceptSet>() {
+      @Override
+      public ConceptSet apply(org.pmiops.workbench.db.model.ConceptSet conceptSet) {
+        if (conceptSet == null) {
+          return null;
+        }
+        ConceptSet result = new ConceptSet()
+          .etag(Etags.fromVersion(conceptSet.getVersion()))
+          .lastModifiedTime(conceptSet.getLastModifiedTime().getTime())
+          .creationTime(conceptSet.getCreationTime().getTime())
+//          .domain(conceptSet.getDomain())
```

Comment:
Is this intentionally commented out? Probably should be including the domain if we can. If it's not working for some reason then please add a TODO for now.

Reply:
Yeah, not working, but also on reflection not sure what good it would do since this is only used for user recent resource, and we don't store the domain (or need to) for that.

Reply:
We don't currently use it, but it is a bit weird to return this resource with missing information. Anyways, I think it's probably fine for now but probably deserves a comment